app/renderer/actions/Inspector.js

```diff
@@ -431,26 +431,39 @@ export function searchForElement (strategy, selector) {
   };
 }
 
-
 export function resetSearchForPORootElement () {
   return (dispatch) => {
-    dispatch({type: SEARCHING_FOR_ROOT_ELEMENTS});
+    dispatch({type: SEARCHING_FOR_ROOT_ELEMENT});
```

Comment:
we through searching the root element of a po to identify the current matching po, so would prefer to use method name from resetSearchForPORootElement to resetSearchForCurrentPOs, and the event also prefer to use: SEARCHING_FOR_CURRENT_PO